app/models/user.rb

```diff
@@ -6,7 +6,12 @@ class User < ApplicationRecord
   attribute :posts_counter, :integer
 
   # Associations
-  has_many :posts, foreign_key: :author_id, dependent: :destroy
-  has_many :comments, dependent: :destroy
+  has_many :posts, foreign_key: 'author_id', dependent: :destroy
+  has_many :comments, foreign_key: 'author_id', dependent: :destroy
   has_many :likes, dependent: :destroy
```

Comment:
- [ ] In response to the previous reviewer's request, it appears that the Like table in the ERD diagram should include a foreign key labeled `author_id`. As a result, I recommend that the user and likes be associated in the following manner: "has_many :likes, foreign_key: 'author_id'". :blush: